samples/drivers/soc_flash_nrf/sample.yaml

```diff
@@ -4,3 +4,20 @@ tests:
   sample.drivers.flash.soc_flash_nrf::
     platform_whitelist: nrf52_pca10040 nrf9160_pca10090 nrf9160_pca10090ns
     tags: flash nrf52
```

Comment:
@nvlsianpu should we add an nrf9160 "tag" here?

Reply:
Looks like yes.

Reply:
OK, I wonder if you could add a separate commit here, to fix that, as well

Reply:
I can add it, no problem.
Just to be sure, this would be the *first* occurrence of such a tag in the repo (I couldn't find any other place which would use "nrf9160" as a tag).
What is the rule here on working with these tags?
Do we really want a tag which will match just one sample, or maybe we should have a separate issue to go through tests&samples compatible with nrf9160 and add them in multiple places at once?

Reply:
Not sure there are generic rules about tags.
This test is an nRF-specific one, so it makes sense to have nrf tags. We can't yet tag with nrf; this is not running on nRF53 (and nRF51), that's why I proposed to tag additionally with nrf91 or nrf9160.

Reply:
btw, we might extend the usage of a tag, that, now, appears to be used on just one sample - there is no big deal with this.

Reply:
Ok then. Tag added.